bench/README.md

```diff
@@ -65,43 +68,123 @@ And open `http://localhost:9090/` to query and monitor the server-side metrics.
 
 ## Headless
 
-To run write/read load tests
-
-```bash
-locust --headless
-```
-
 To run write load tests
 
-```bash
-locust --headless DCLWriteUser
-```
+* Open the docker-compose.yml and update field `command` for `master` and `worker`.  
+    e.g.: for `master`
+    ```yml
+    command: -f /dcl/bench/locustfile.py --headless WriteModelLoadTest --master -H http://master:8089 
+    ```
+    e.g.: for `worker`:
+    ```yml
+    command: -f /dcl/bench/locustfile.py --headless WriteModelLoadTest --worker --master-host master
+    ```
+
+* Run docker-compose.yml
+    ```bash
+    docker-compose up --scale worker=<workers-count>
+    # The workers run your Users and send back statistics to the master. The master instance doesn't run any Users itself. Both the master and worker machines must have a copy of the locustfile when running Locust distributed.
+    ```
+    `<workers-count>`- number of machine.
```

Comment:
Mention that number of users should equal to number of workers for write tests

Reply:
Done! ✅